jekyll/_cci2/upgrading-circleci-machine-runner-on-cloud.adoc

```diff
@@ -11,7 +11,7 @@ contentTags:
 :toc: macro:
 :toc-tittle:
 
-CAUTION: **Self-hosted runner version 1.0 will sunset on July 27, 2023,** and will be temporarily unavailable on June 28, 2023. **Update to 1.1.**
+CAUTION: **Self-hosted runner version 1.0 is temporarily unavailable today starting from 12:00PM EST.** Version 1.0 will permanently sunset on July 27, 2023. **Update to 1.1.**
```

Comment:
Just wondering why the snippet is not used here?

Reply:
Ah should've included that in the PR description - the "Update to 1.1" link actually directs to this page. So the snippet is used everywhere else whereas it's hardcoded here because it does not have a link.

Reply:
Oh I see! Thanks!!